tests/test_twiss.py

```diff
@@ -1997,3 +1997,59 @@ def test_twiss_collective_end_is_len():
     t2 = line2.twiss4d(betx=1,bety=1,include_collective=True)
 
     xo.assert_allclose(t.betx, t2.betx, atol=1e-12, rtol=0)
+
+
+def test_twiss_compute_coupling_elements_edwards_teng():
```

Comment:
We need to test what happens when `reverse` is True.
Alternatively we should forbid reverse=True when Edw Teng coefficient are required. If we do the second, we should also remove these columns in the method `TwissTable.reverse(...).